docs/wiki/Upgrading.md

```diff
@@ -1,24 +1,51 @@
-# Upgrade Process
+# Updating Eventum versions
 
 One of our objectives is to make upgrading from an earlier release as
 painless as possible, and we provide scripts that should bring your
 existing Eventum installation up-to-date.
 
+Eventum supports sequential upgrade path using released minor versions.
+
+In other words, upgrade to minor version separately, you can skip patch
+versions in minor versions.
```

Comment:
I would say like this: "To explain more clearly one may skip all patch versions and upgrade to the next minor version."

Reply:
454f4f39a